ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -893,8 +894,12 @@ export default defineComponent({
       setEmptyMhrTransfer(initMhrTransfer())
       setEmptyMhrTransportPermit(initTransportPermit())
 
+      const isFrozenDueToTransportPermit =
+        getMhrInformation.value.frozenDocumentType === UnitNoteDocTypes.TRANSPORT_PERMIT
+
       // Set baseline MHR Information to state
-      await parseMhrInformation(isFrozenMhr.value)
+      // Do not parse Transfer details when frozen doc type is Transport Permit - bug 19780
```

Comment:
You want to eternalize the ticket number in the code comment?

Reply:
To reference the issue, for tricky bugs it is helpful. We can look up the commit and track it that way, but sometimes it gets overwritten by upgrades or other tickets.

Reply:
Well it sets a new precedent, were going to do this for all bugs? Tricky bugs? Whats a tricky bug? 

Either way, i disagree with this when your editor can give you this information with a click but whatever.

Reply:
A tricky bug is when it takes 3 people to figure out the issue, and come up with a workaround like this when there is no straightforward fix. It's ok to reference bugs. And I said said, sometime it gets overwritten.

Reply:
What gets overwritten? Git History tied to commits, tied to PRs? I don't think so